<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>831</width>
      <height>471</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home"/>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="send">
     <widget class="QLineEdit" name="addr">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>311</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>адрес</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="summ">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>311</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>сумма</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Отправить</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="hash">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>761</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="note">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>311</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>примечание (видно только вам)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>БАЛАНС</string>
    </property>
   </widget>
   <widget class="QLabel" name="balance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="show_bal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать баланс</string>
    </property>
   </widget>
   <widget class="QPushButton" name="receive_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
   <widget class="QLabel" name="hello">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>История</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contacts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Контакты</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
